src/util.h

```diff
@@ -539,4 +538,36 @@ GIT_INLINE(double) git__timer(void)
 
 extern int git__getenv(git_buf *out, const char *name);
 
+#ifdef WIN32
+#define S_ISREG(m) (((m) & _S_IFMT) == _S_IFREG)
```

Comment:
This is only temporary, but I didn't know where to put it (and VS2010 actually doesn't like it being redefined).

Reply:
Probably safer to:
```
#ifndef S_ISREG
# define S_ISREG...
#endif
```

Reply:
Actually, I wanted to move it in our compat `posix` headers for Windows (with the define guard), but I'm not sure where/which one I should use.